org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/report/HTMLExtension.java

```diff
@@ -203,6 +204,16 @@ public String getFullPathStyle() {
 			theme = theme + "/";
 		String resFile = theme + "css/report.css";
 		
+		// Support to Parse Context Variable
+		if (resFile.contains("@"))
+		{
+			resFile = Env.parseContext(Env.getCtx(), 0, resFile, false);
+			if (Util.isEmpty(resFile))
```

Comment:
this is probably undesirable, should at least log a warning for it.

Reply:
Added log with warning.